src/plugins/usage_collection/server/plugin.ts

```diff
@@ -59,11 +59,6 @@ export interface UsageCollectionSetup {
   getCollectorByType: <TFetchReturn, ExtraOptions extends object>(
     type: string
   ) => Collector<TFetchReturn, ExtraOptions> | undefined;
-  /**
-   * Returns if all the collectors are ready to fetch their reported usage.
-   * @internal: telemetry use
-   */
-  areAllCollectorsReady: () => Promise<boolean>;
```

Comment:
Should we run the API Docs extract to update them?

Reply:
It is updated everywhere 🤔